Week3/homework/step2-3.js

```diff
@@ -32,6 +39,12 @@ function repeatStringNumTimesWithDoWhile(str, num) {
   let result = '';
 
   // Replace this comment and the next line with your code
+  if (num > 0) {
```

Comment:
Any reason why you added this `if` check here in the `do-while` loop and not `for` or `while` loop?

Reply:
Thanks for your feedback .when i have checked without `if ` ( npm run test-week3 ) , had an error that -

> step2-3 with do-while-loop › num = -2
so i added this `if` check.